### PR TITLE
refactor: disable atomic styles in generate stories

### PR DESCRIPTION
**Pull request by TrySound**
Ref https://github.com/webstudio-is/webstudio/issues/3788

Here switched back to regular (non-atomic) styles
in generated stories in radix sdk.

### CODE REVIEW

File: packages/sdk-components-react-radix/src/__generated__/accordion.stories.tsx [hunk 1]
```diff
@@ -187,105 +169,106 @@ html {margin: 0; display: grid; min-height: 100%}
...
+  :where(address.w-box) {
     box-sizing: border-box;
     border-top-width: 1px;
     border-right-width: 1px;
     border-bottom-width: 1px;
     border-left-width: 1px;
     outline-width: 1px
   }
-  article:where([data-ws-component="Box"]) {
+  :where(article.w-box) {
```
Comment (kof) [inline packages/sdk-components-react-radix/src/__generated__/accordion.stories.tsx:191]: do you think it makes sense to just have a simple .w-box here?

**Review by kof — APPROVED** [review]

File: packages/sdk-components-react-radix/src/__generated__/accordion.stories.tsx (hunk 1 above)
Comment (TrySound) [reply packages/sdk-components-react-radix/src/__generated__/accordion.stories.tsx:191]: Nope. We use :where to reset specificity and let stateless styles override hover. Same like we do with a:visited {} styles.

Comment (TrySound) [reply packages/sdk-components-react-radix/src/__generated__/accordion.stories.tsx:191]: Same effect would give be using `@layer preset {}`

Comment (kof) [reply packages/sdk-components-react-radix/src/__generated__/accordion.stories.tsx:191]: right, forgot about it

Comment (kof) [reply packages/sdk-components-react-radix/src/__generated__/accordion.stories.tsx:191]: ok but why do we need the tag selector as well?

**Pull request merged**